notes/stv/20220708-01-1Tb-disk-changes-test.txt

```diff
@@ -0,0 +1,788 @@
+# Check disks
+
+ls -al /var/hadoop/
+total 8
+drwxr-xr-x.  2 fedora fedora 4096 Jul  8 09:49 .
+drwxr-xr-x. 20 root   root   4096 Jul  8 09:51 ..
+lrwxrwxrwx.  1 fedora fedora   27 Jul  8 09:47 data -> /mnt/cinder/vdd/hadoop/data
+lrwxrwxrwx.  1 fedora fedora   26 Jul  8 09:48 logs -> /mnt/local/vdb/hadoop/logs
+lrwxrwxrwx.  1 fedora fedora   27 Jul  8 09:49 temp -> /mnt/cinder/vde/hadoop/temp
+
+df -h /var/hadoop/data/
+Filesystem      Size  Used Avail Use% Mounted on
+/dev/vdd        1.0T  648M 1022G   1% /mnt/cinder/vdd
+
+df -h /var/hadoop/temp/
+Filesystem      Size  Used Avail Use% Mounted on
+/dev/vde        1.0T   17M 1022G   1% /mnt/cinder/vde
+
+df -h /var/hadoop/logs/
+Filesystem      Size  Used Avail Use% Mounted on
+/dev/vdb        178G   62M  168G   1% /mnt/local/vdb
+
+
+
+# -----------------------------------------------------
+# Create 3 test users
+#[root@ansibler]
+
+    count=3
+
+    testernames=()
+
+    for i in $(seq $((count+1)))
+    do
+        testernames+=($(pwgen 12 1))
+    done
+
+    createarrayusers \
+        ${testernames[@]} \
+    | tee /tmp/testusers.json \
+    | jq '[ .users[].shirouser.name ]'
+
+# Success
+
+
+	
+# -----------------------------------------------------
+# Run our tests
+
+    time \
+        /deployments/hadoop-yarn/bin/run-tests.sh \
+            "${cloudname:?}" \
+            "${configname:?}" \
+            "${test_level:?}"  \
+	     ${concurrent:?}  \
+	     ${num_users:?}  \
+        | tee /tmp/run-tests-quick.log
+	
+	# Cancel and run again for better output..
+	
+
+-------------------------------------
+# Run single user test
+# Users: 1
+# Delay_start: 0
+# Delay_notebook: 0
+# Delete Notebooks: True
+
+
+cat > /tmp/testprog.py << EOF
+import sys
+from aglais_benchmark import AglaisBenchmarker
+AglaisBenchmarker(
+    "/deployments/zeppelin/test/config/quick.json",
+    "/tmp/testusers.json",
+    "/tmp/",
+    "http://128.232.227.196:8080",
+    False
+    ).run(
+        concurrent=False,
+        users=1,
+        delay_start=0,
+        delay_notebook=0,
+        delete=True
+        )
+EOF
+
+python3 /tmp/testprog.py | tee /tmp/test-results.txt
+
+    
+[{
+	"name": "GaiaDMPSetup",
+	"result": "ERROR",
+	"outputs": {
+		"valid": true
+	},
+	"messages": [],
+	"time": {
+		"result": "SLOW",
+		"elapsed": "53.95",
+		"expected": "45.00",
+		"percent": "19.88",
+		"start": "2022-07-08T11:06:41.069800",
+		"finish": "2022-07-08T11:07:35.016634"
+	},
+	"logs": "Fail to execute line 5: assert all(item in actual_tables for item in expected_tables)\nTraceback (most recent call last):\n  File \"/tmp/1657278409538-0/zeppelin_python.py\", line 158, in <module>\n    exec(code, _zcUserQueryNameSpace)\n  File \"<stdin>\", line 5, in <module>\nAssertionError"
+}, {
+	"name": "Mean_proper_motions_over_the_sky",
+	"result": "PASS",
+	"outputs": {
+		"valid": true
+	},
+	"messages": [],
+	"time": {
+		"result": "SLOW",
+		"elapsed": "156.23",
+		"expected": "55.00",
+		"percent": "184.06",
```

Comment:
This is what we need top look at.
Is this because we are using Cinder volumes for `/var/hadoop/temp`, or is this caused by something else.